### ISSUE TITLE
optics-extra blows up with GHC 9

### ISSUE THREAD

**Issue by kozross** [well-typed/optics#402]
```
[21 of 24] Compiling Optics.Passthrough ( src/Optics/Passthrough.hs, dist/build/Optics/Passthrough.o, dist/build/Optics/Passthrough.dyn_o )

src/Optics/Passthrough.hs:19:17: error:
    • Couldn't match type: forall (f :: * -> *).
                           Functor f =>
                           (a -> f b) -> s -> f t
                     with: (a -> (r, b)) -> s -> (r, t)
      Expected: Optic An_Iso is s t a b
                -> (a -> (r, b)) -> s -> (ViewResult An_Iso r, t)
        Actual: Optic An_Iso is s t a b -> LensVL s t a b
    • In the expression: toLensVL
      In an equation for ‘passthrough’: passthrough = toLensVL
      In the instance declaration for ‘PermeableOptic An_Iso r’
    • Relevant bindings include
        passthrough :: Optic An_Iso is s t a b
                       -> (a -> (r, b)) -> s -> (ViewResult An_Iso r, t)
          (bound at src/Optics/Passthrough.hs:19:3)
   |
19 |   passthrough = toLensVL
   |                 ^^^^^^^^

src/Optics/Passthrough.hs:23:17: error:
    • Couldn't match type: forall (f :: * -> *).
                           Functor f =>
                           (a -> f b) -> s -> f t
                     with: (a -> (r, b)) -> s -> (r, t)
      Expected: Optic A_Lens is s t a b
                -> (a -> (r, b)) -> s -> (ViewResult A_Lens r, t)
        Actual: Optic A_Lens is s t a b -> LensVL s t a b
    • In the expression: toLensVL
      In an equation for ‘passthrough’: passthrough = toLensVL
      In the instance declaration for ‘PermeableOptic A_Lens r’
    • Relevant bindings include
        passthrough :: Optic A_Lens is s t a b
                       -> (a -> (r, b)) -> s -> (ViewResult A_Lens r, t)
          (bound at src/Optics/Passthrough.hs:23:3)
   |
23 |   passthrough = toLensVL
   |                 ^^^^^^^^
cabal: Failed to build optics-extra-0.3
```

GHC version is 9.0.1

**Comment by arybczak** [comment]
It's fixed in master, will be fixed by 0.4 (#375).

**Comment by kozross** [comment]
As this is now released, we can close this issue. Thanks!
